store/server/features/dashboard/recognitions/queries.ts

```
@@ -0,0 +1,83 @@
+    const headers = {
+      Authorization: `Bearer ${token}`,
+      tenantId: tenantId,
+    };
+
+    return await crudRequest({
+      url: `${ORG_DEV_URL}/recognition/super-star-recognition`,
+      method: 'GET',
+      headers,
+    });
+  } catch (error) {
+    throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)